tests/test.py

```diff
@@ -0,0 +1,5 @@
+from trace_dkey import trace
```

Comment:
use pytest, pytest has fixture, which will be handy to write unit test for these kind of package